src/components/Timeline/Controls.tsx

```diff
@@ -219,14 +281,25 @@ export const Controls: FC<TimelineControlsProps> = memo(({
       </Elem>
 
       <Elem name="group" tag={Space} size="small">
-        <TimeDisplay
-          currentTime={currentTime}
-          duration={duration}
-          length={length}
-          position={position}
-          framerate={frameRate}
-          formatPosition={formatPosition}
-        />
+        {!isFF(FF_DEV_2715) ? (
+          <TimeDisplay
+            currentTime={currentTime}
+            duration={duration}
+            length={length}
+            position={position}
+            framerate={frameRate}
+            formatPosition={formatPosition}
+          />
+        ) : (
+          <TimeDurationControl
+            startTime={0}
+            endTime={duration}
+            minTime={0}
+            maxTime={duration}
+            currentTime={currentTime}
+            onChangeStartTime={onTimeUpdateChange}
+          />
+        )}
```

Comment:
Probably should swap this around to being in the positive tense as it reads a bit easier.
```suggestion
        {isFF(FF_DEV_2715) ? (
          <TimeDurationControl
            startTime={0}
            endTime={duration}
            minTime={0}
            maxTime={duration}
            currentTime={currentTime}
            onChangeStartTime={onTimeUpdateChange}
          />
        ) : (
          <TimeDisplay
            currentTime={currentTime}
            duration={duration}
            length={length}
            position={position}
            framerate={frameRate}
            formatPosition={formatPosition}
          />
        )}
```